Simplenote/AuthViewController+Swift.swift

```diff
@@ -281,9 +333,16 @@ extension AuthViewController {
         SPTracker.trackWPCCButtonPressed()
     }
 
+    @objc
+    func performLogInWithCode() {
+        Task {
+            await loginWithCode(username: state.username, code: state.code)
+        }
+    }
+
     @IBAction
     func handleNewlineInField(_ field: NSControl) {
-        if field.isEqual(passwordField.textField) {
+        if shouldHandleNewLine(in: field) {
```

Comment:
Not sure if we need this API (there are just 3 TextFields and we do the same for all 3 of them?)

Reply:
I think it is cleaner to have the separate method, but am happy to not